content/_data/changelogs/weekly.yml

```diff
@@ -14157,6 +14157,169 @@
         authors:
           - Wadeck
 
+  - version: '2.335'
+    date: 2022-02-13
+    changes:
+      - type: TODO
+        category: developer
+        pull: 6262
+        authors:
+          - dependabot[bot]
+          - basil
+        pr_title: 'Winstone 5.22: Bump Jetty from 9.4.43.v20210629 to 9.4.45.v20220203'
+        message: |-
```

Comment:
Not sure how this needs to be formatted in order to look as intended in the changelog.
Plus, this is a "developer"-category entry, so needs to be added as the last entry

Reply:
Will need to be moved into the correct location and have additional links after the release is complete.

@MarkEWaite action item after the weekly release has been delivered.